addons/debugger/timing/ui-components/TableRows.js

```diff
@@ -0,0 +1,95 @@
+import LogView from "../../log-view.js"; // Assuming LogView is imported from elsewhere
+
+function createInfoElement(text) {
+  const elem = document.createElement("span");
+  elem.textContent = text;
+  return elem;
+}
+
+class TableRows extends LogView {
+  constructor(config, debug, msg, tableHeader) {
+    super();
+    this.config = config;
+    this.totalTime = null;
+    this.minTimerTime = null;
+    this.debug = debug;
+    this.tableHeader = tableHeader;
+
+    this.placeholderElement.textContent = msg("timing-no-timers");
+  }
+
+  getRowValues(timer) {
+    const rowValues = {
+      label: timer.label,
```

Comment:
Is there anything that can be done about long labels causing overlapping text? Maybe it should go to an ellipsis
![image](https://github.com/user-attachments/assets/885d81be-dbb6-4641-9dc5-03b3551ea199)